webpack/dev.config.js

```diff
@@ -8,7 +8,7 @@ const customPath = path.join(__dirname, './customPublicPath');
 const hotScript = 'webpack-hot-middleware/client?path=/__webpack_hmr&dynamicPublicPath=true';
 
 const baseDevConfig = () => ({
-  devtool: 'eval-cheap-module-source-map',
+  devtool: 'eval-source-map',
```

Comment:
Why not use <!-- devtool: "eval-cheap-module-source-map"  i understand it doesn't provide column's but it's much faster, right? -->